src/condor_daemon_client/dc_schedd.cpp

```diff
@@ -2104,3 +2104,277 @@ DCSchedd::requestImpersonationTokenAsync(const std::string &identity,
 	}
 	return true;
 }
+
+// TODO: how do we make this global??
```

Comment:
Make what global?

Reply:
The subsequent anonymous enum is actually a redefinition (from the `CondorQ` object header?); the two definitions should be unified and made available for the whole codebase.